shared/constants/platform-notifications.ts

```diff
@@ -0,0 +1,149 @@
+  ARBITRUM: `${parseInt(CHAIN_IDS.ARBITRUM, 16)}`,
+  AVALANCHE: `${parseInt(CHAIN_IDS.AVALANCHE, 16)}`,
+  LINEA: `${parseInt(CHAIN_IDS.LINEA_MAINNET, 16)}`,
+};
+
+export const SUPPORTED_CHAINS = [
+  chains.ETHEREUM,
+  chains.OPTIMISM,
+  chains.BSC,
+  chains.POLYGON,
+  chains.ARBITRUM,
+  chains.AVALANCHE,
+  chains.LINEA,
+];
+
+export interface Trigger {
+  supported_chains: (typeof SUPPORTED_CHAINS)[number][];
+}
+
+export const TRIGGERS: Partial<Record<TRIGGER_TYPES, Trigger>> = {
+  [TRIGGER_TYPES.METAMASK_SWAP_COMPLETED]: {
+    supported_chains: [
+      chains.ETHEREUM,
+      chains.OPTIMISM,
+      chains.BSC,
+      chains.POLYGON,
+      chains.ARBITRUM,
+      chains.AVALANCHE,
+    ],
+  },
+  [TRIGGER_TYPES.ERC20_SENT]: {
+    supported_chains: [
+      chains.ETHEREUM,
+      chains.OPTIMISM,
+      chains.BSC,
+      chains.POLYGON,
+      chains.ARBITRUM,
+      chains.AVALANCHE,
+      chains.LINEA,
+    ],
+  },
+  [TRIGGER_TYPES.ERC20_RECEIVED]: {
+    supported_chains: [
+      chains.ETHEREUM,
+      chains.OPTIMISM,
+      chains.BSC,
+      chains.POLYGON,
+      chains.ARBITRUM,
+      chains.AVALANCHE,
+      chains.LINEA,
+    ],
+  },
+  [TRIGGER_TYPES.ERC721_SENT]: {
+    supported_chains: [chains.ETHEREUM, chains.POLYGON],
+  },
+  [TRIGGER_TYPES.ERC721_RECEIVED]: {
+    supported_chains: [chains.ETHEREUM, chains.POLYGON],
+  },
+  [TRIGGER_TYPES.ERC1155_SENT]: {
+    supported_chains: [chains.ETHEREUM, chains.POLYGON],
+  },
+  [TRIGGER_TYPES.ERC1155_RECEIVED]: {
+    supported_chains: [chains.ETHEREUM, chains.POLYGON],
+  },
+  [TRIGGER_TYPES.ETH_SENT]: {
+    supported_chains: [
+      chains.ETHEREUM,
+      chains.OPTIMISM,
+      chains.BSC,
+      chains.POLYGON,
+      chains.ARBITRUM,
+      chains.AVALANCHE,
+      chains.LINEA,
+    ],
+  },
+  [TRIGGER_TYPES.ETH_RECEIVED]: {
+    supported_chains: [
+      chains.ETHEREUM,
+      chains.OPTIMISM,
+      chains.BSC,
+      chains.POLYGON,
+      chains.ARBITRUM,
+      chains.AVALANCHE,
+      chains.LINEA,
+    ],
+  },
+  [TRIGGER_TYPES.ROCKETPOOL_STAKE_COMPLETED]: {
+    supported_chains: [chains.ETHEREUM],
+  },
+  [TRIGGER_TYPES.ROCKETPOOL_UNSTAKE_COMPLETED]: {
+    supported_chains: [chains.ETHEREUM],
+  },
+  [TRIGGER_TYPES.LIDO_STAKE_COMPLETED]: {
+    supported_chains: [chains.ETHEREUM],
+  },
+  [TRIGGER_TYPES.LIDO_WITHDRAWAL_REQUESTED]: {
+    supported_chains: [chains.ETHEREUM],
+  },
+  [TRIGGER_TYPES.LIDO_WITHDRAWAL_COMPLETED]: {
+    supported_chains: [chains.ETHEREUM],
+  },
+};
+
+export enum TRIGGER_DATA_TYPES {
+  DATA_FEATURE_ANNOUNCEMENT = 'data_feature_announcement',
+  DATA_METAMASK_SWAP_COMPLETED = 'data_metamask_swap_completed',
+  DATA_STAKE = 'data_stake',
+  DATA_LIDO_STAKE_READY_TO_BE_WITHDRAWN = 'data_lido_stake_ready_to_be_withdrawn',
+  DATA_ETH = 'data_eth',
+  DATA_ERC20 = 'data_erc20',
+  DATA_ERC721 = 'data_erc721',
+  DATA_ERC1155 = 'data_erc1155',
+}
+
+export const CHAIN_SYMBOLS = {
```

Comment:
This const seems to serve a similar purpose as the `CURRENCY_SYMBOLS` or `CHAINLIST_CURRENCY_SYMBOLS_MAP` const in `shared/constants/network.ts`. However, some of the symbols here are different from the symbols in that file (for example, the optimism symbol is different).

I might make sense to import the symbols from `shared/constants/network.ts` to this file, to make sure we are using the same symbols throughout the application.

(Which are the correct symbols is a separate question, but we should make sure we are consistent across the application)

Reply:
Yes, you're absolutely right. I've updated the file!

Reply:
I have remove this file :)